pages.es/common/tgpt.md

```diff
@@ -0,0 +1,37 @@
+# tgpt
+
+> Habla con un chatbot de IA sin necesidad de claves API.
+> Proveedores disponibles: `openai`, `opengpts`, `koboldai`, `phind`, `llama2`, `blackboxai`.
+> Más información: <https://github.com/aandrew-me/tgpt>.
+
+- Chatea con el proveedor por defecto (GPT-3.5-turbo):
+
+`tgpt "{{prompt}}"`
+
+- Inicia el modo interactivo [m]ultilínea:
+
+`tgpt --multiline`
+
+- Genera [i]mágenes y las guarda en el directorio actual:
+
+`tgpt --image "{{prompt}}"`
+
+- Genera [c]ódigo con el proveedor por defecto (GPT-3.5-turbo):
+
+`tgpt --code "{{prompt}}"`
+
+- Chatea con un proveedor específico [q]uietamente (sin animaciones):
+
+`tgpt --provider {{openai|opengpts|koboldai|phind|llama2|blackboxai}}} --quiet --whole "{{prompt}}"`
+
+- Genera y ejecuta comandos [s]hell utilizando un proveedor específico (con un prompt de confirmación):
+
+`tgpt --provider {{llama2}}} --shell "{{prompt}}"`
+
+- Pregunta con una clave de API, modelo, longitud máxima de respuesta, temperatura y `top_p` (necesario cuando se utiliza el proveedor `openai`):
+
+`tgpt --provider openai --key "{{api_key}}" --model "{{gpt-3.5-turbo}}" --max-length {{10}} --temperature {{0.7}} --top_p {{0.9}} "{{prompt}}"`
```

Comment:
```suggestion
`tgpt --provider openai --key "{{api_key}}" --model "{{gpt-3.5-turbo}}" --max_length {{10}} --temperature {{0.7}} --top_p {{0.9}} "{{prompt}}"`
```

I think the words in the option strings are separated by an underscore rather than a hyphen[^1][^2].

[^1]: https://github.com/aandrew-me/tgpt?tab=readme-ov-file#usage-
[^2]: https://github.com/aandrew-me/tgpt/blob/main/main.go#L63